rest-api/src/test/java/gov/cms/qpp/conversion/api/acceptance/CpcApiAcceptance.java

```diff
@@ -48,7 +50,7 @@
 
 	@BeforeAll
 	static void setUp() {
-		String kmsKey = "arn:aws:kms:us-east-1:850094054452:key/8b19f7e9-b58c-4b7a-8162-e01809a8a2e9";
+		String kmsKey = System.getenv(Constants.TEST_KMS_KEY_ENV_VARIABLE);
```

Comment:
What about using the `EnvironmentHelper` class and calling `getOrDefault`?

Reply:
@halprin it's essentially the same thing. I do prefer this as I think it's more clear. I'd just be moving around the Arn.